Bojzen-I-Mitten/Stort-spelprojekt
# Match/aim

GUI now have an aim circle and a bar that increase when the player charges a throw.

To test:
- Update SVN
- Check so that the Chad.prefab have the correct script:
-- ChadHUD.cs
-- chadControls.cs
- Use the superUltraMatchScene
- If the chad prefab was incorrect. Add the new chad to the scene.
- Host a game.
- Pick up the ball and try throwing it.
-- Should see an aim and a bar that expands when charging the ball.
